README.md

```diff
@@ -35,6 +35,9 @@ Supported props
 | touch | boolean | true | If touch gestures should be enabled |
 | touchHandleWidth | number | 20 | Width in pixels you can start dragging from the edge when the sidebar is closed. |
 | dragToggleDistance | number | 30 | Distance the sidebar has to be dragged before it will open/close after it is released. |
+| pullRight | boolean | false | Place the sidebar on the right |
+| topSidebar | number | 0 | Sidebar distance from the top. |
```

Comment:
Would you accept it if I add :warning:  *Use with caution* :warning:  in the README :smile: ?!

This feature is useful on my project and it is working like a charm...